auth-server/src/services/auth.ts

```diff
@@ -594,12 +588,6 @@ export default class AuthService {
 			throw new UserInputError(messages.EMAIL_NOT_FOUND);
 		}
 
```

Comment:
Similar to what was done with the previous flow, we should delete any verification token for this user.

Reply:
Cannot be done with redis design. We are saving EVT-<token>: email as key value pair. Will not be able to find user tokens.

Reply:
Right, I figured it out. Also I can't think about an attack vector for this.